### PR TITLE
Update chipster info

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Changes in the chipster config

----

**Related issue :**

### CODE REVIEW

**Review by glarocca — DISMISSED** [review]

File: source/aod/chipster.rst [hunk 1]
```diff
@@ -19,9 +19,8 @@ The Chipster testbed configured at `CESGA <https://www.cesga.es/>`_ offers:
 * 8 vCPU cores,
 * 32GB of RAM,
 * 1TB of block storage in /data,
-* Software and tools (v3.12.3) are in available under the /cvmfs/chipster.egi.eu/
-  partition,
-* Chipster (v3.12.3).
+*  Software and tools are in available under the ``/cvmfs/tools_*`` partition,
```
Comment (enolfc) [inline source/aod/chipster.rst]:
```suggestion
* Software and tools are in available under the ``/cvmfs/tools_*`` partition,
```

**Review by enolfc — COMMENTED** [review]
Re-build!

**Comment by enolfc** [comment]
@glarocca can you re-approve? There was some issue with travis that prevented the merge

**Comment by gwarf** [comment]
@enolfc please rebase and then we can approve/close this.

**Comment by enolfc** [comment]
No longer relevant as this content is already merged in main EGI docs

**Pull request closed without merging**